lib/hermod/version.rb

```diff
@@ -1,3 +1,3 @@
 module Hermod
-  VERSION = "2.4.1"
```

Comment:
Should this be `2.5.0` to match what's mentioned in the changelog?

Reply:
Done